.travis.yml

```diff
@@ -4,4 +4,12 @@ services:
     - docker
   
 script:
-    - ./test-build.sh
+    - ./test-build.sh $NODE_VERSION
```

Comment:
Oh! I didn't realize there was a time-out. In that case leave it and we can revisit when there are pre-built binaries.